db/schema.rb

```diff
@@ -104,6 +112,17 @@
     t.string "source"
   end
 
+  create_table "appeals_api_status_updates", force: :cascade do |t|
+    t.string "from"
+    t.string "to"
+    t.string "appeal_type"
```

Comment:
When we still thought EvidenceSubmission would have statuses under our control, I figured this table would store those as well. What're your thoughts on making this a little more generic so that it can track any model w/ a status field we want?

Reply:
I have no issue with that. What do you have in mind?